Package@swift-6.swift

```diff
@@ -207,7 +207,11 @@ extension Target {
         .atomics
       ],
       path: "Sources/GRPCCore",
-      swiftSettings: [._swiftLanguageMode(.v5), .enableUpcomingFeature("ExistentialAny")]
+      swiftSettings: [
+        ._swiftLanguageMode(.v5),
+        .enableUpcomingFeature("ExistentialAny"),
+        .enableUpcomingFeature("InternalImportsByDefault")
```

Comment:
If we have to enable this, then presumably `public` is the default for Swift 6?

Reply:
Yes: the default will become `internal` in a future language mode ([see proposal](https://github.com/swiftlang/swift-evolution/blob/main/proposals/0409-access-level-on-imports.md#default-import-access-level)).